labgrid/util/readmixin.py

```diff
@@ -0,0 +1,77 @@
+from .timeout import Timeout
+from pexpect import EOF, TIMEOUT
```

Comment:
```suggestion
from pexpect import EOF, TIMEOUT

from .timeout import Timeout
```
nitpick: wrong import order